src/test/java/org/commcare/formplayer/tests/BaseTestClass.java

```diff
@@ -828,6 +828,7 @@ <T> T sessionNavigateWithSelectedValues(String[] selections, String testName, St
         sessionNavigationBean.setSelections(selections);
         sessionNavigationBean.setSelectedValues(selectedValues);
         sessionNavigationBean.setWindowWidth(windowWidth);
+        sessionNavigationBean.setLocale(locale);
```

Comment:
How is this working given `sessionNavigationBean` does no longer have `setLocale` method ?